src/tests/test_circumcision.py

```diff
@@ -0,0 +1,435 @@
+from datetime import date, timedelta
+from math import sqrt
+from time import perf_counter
+
+import numpy as np
+
+import hivpy.column_names as col
+from hivpy.common import SexType
+from hivpy.population import Population
+
+
+# Find average birth circumcicion time: init all at once vs
+# partial init followed by average time of updates per time step
+def test_birth_circumcision_timing():
```

Comment:
I think as long as we have the results of the timing recorded, we don't need to include the timing in the unit tests right now as it's not something that's likely to change as we develop and we don't want to necessarily run it every time as it takes just under a minute on my machine, a considerable fraction of the total test time so far